cmd/run.go

```diff
@@ -102,6 +102,8 @@ func bindRunFlags(cmd *cobra.Command, config *app.Config) {
 	cmd.Flags().StringVar(&config.ExecutionEngineAddr, "execution-client-rpc-endpoint", "", "The address of the execution engine JSON-RPC API.")
 	cmd.Flags().StringSliceVar(&config.Graffiti, "graffiti", nil, "Comma-separated list or single graffiti string to include in block proposals. List maps to validator's public key in cluster lock. Appends \"OB<CL_TYPE>\" suffix to graffiti. Maximum 28 bytes per graffiti.")
 	cmd.Flags().BoolVar(&config.GraffitiDisableClientAppend, "graffiti-disable-client-append", false, "Disables appending \"OB<CL_TYPE>\" suffix to graffiti. Increases maximum bytes per graffiti to 32.")
+	cmd.Flags().StringVar(&config.VCTLSCertFile, "vc-tls-cert-file", "", "The path to the TLS certificate file used by the Validator Client for secure communication.")
+	cmd.Flags().StringVar(&config.VCTLSKeyFile, "vc-tls-key-file", "", "The path to the TLS key file used by the Validator Client for secure communication.")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&config.VCTLSCertFile, "vc-tls-cert-file", "", "The path to the TLS certificate file used by charon for secure communication with the validator client.")
	cmd.Flags().StringVar(&config.VCTLSKeyFile, "vc-tls-public-key-file", "", "The path to the TLS public key file used by the validator client for secure communication with charon.")
```


We use a mix of upper and lower case in our cli text but majority lower so going with that. Added a little more color on what the cert and pubkey files are to help them be more easily understood for someone unfamiliar.